index.html

```diff
@@ -114,6 +113,8 @@ <h3>Installing Hg-Git the easy way:</h3>
           comes with hg-git (and Dulwich), though it 
           <a href="https://tortoisehg.readthedocs.io/en/latest/nonhg.html#hg-git-git">still needs to be enabled</a>
           as shown below (or through the TortoiseHg settings).
+          This currently only <a href="https://bitbucket.org/tortoisehg/thg/issues/5140/hg-git-abort-no-module-named-selectors">
+          works with TortoiseHg up to version 4.5.2.</a>
```

Comment:
Is this still the case with hg-git 0.8.13?

Reply:
This applies to the hg-git version bundled with TortoiseHg. hg-git 0.8.13 is not included in the latest TortoiseHg version 5.0.2.